tests/clash-testsuite.cabal

```diff
@@ -12,6 +12,8 @@ maintainer:          Christiaan Baaij <christiaan.baaij@gmail.com>
 copyright:           Copyright © 2015 University of Twente
 category:            Testing
 build-type:          Simple
+data-files:
+  tcl/clash_namespace.tcl
```

Comment:
Could we make the name of the directory more generic? `global-files`? Otherwise we'll need another directory when we have a different reason to add a file.

Reply:
I figured it'd make sense to put all `tcl` sources in one directory. But if you prefer to put all `data-files` in the same directory I suppose that could work :thinking:

Reply:
I think Peter would like `data-files/tcl`

Reply:
in case we ever want something else than `tcl`.

Reply:
Peter approved IRL.